package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "build": "remix build",
     "dev": "remix dev",
-    "start": "cross-env NODE_ENV=production netlify dev",
+    "start": "netlify serve",
```

Comment:
Using ntl serve as it makes more sense in this context.

Reply:
I'd normally expect `start` to run `dev`, but I guess it could go either way. Adding a `serve` command could be good.

Reply:
Was going with your suggestion from the other day unless I was dreaming. 🙃 . Using serve avoids running build explicitly which is nice and ntl dev still gives us dev mode

Reply:
What I meant was that if people wanted the previous behaviour (i.e. running the prod server) then they could use ntl serve instead